server/src/main/groovy/com/insilico/dmc/ingester/Ingester.groovy

```diff
@@ -240,4 +241,19 @@ abstract class Ingester {
             }
         }
     }
+
+//        xmlData = xmlData.findAll("<sec id=\"(.+?)\" sec-type=\"(.+?)\">")
+//    final Pattern pattern = Pattern.compile("<sec id=\"(.+?)\" sec-type=\"(.+?)\">", Pattern.DOTALL)
+    static String convertAllSec(String input) {
```

Comment:
This might be a better way to handle the comments and other issues with genetics pubs.